types/lib/client/index.d.ts

```diff
@@ -37,7 +37,7 @@ declare class Client {
     _supscriptions: Map<string, () => void>;
     _sentPending: Promise<void>;
     get key(): any;
-    get id(): string;
+    get id(): any;
```

Comment:
Saw the `any` type here. Any reason for that? Might be better with a specific type if possible. Thoughts?

Reply:
It also seams that this is not related to the scope of this PR. 
Maybe could be a good idea to move this code to another PR.